### ISSUE TITLE
multi-core rendering

### ISSUE THREAD

**Issue by BugBerry202** [JargeZ/vhs#117]
there was an issue made 3 years ago about slow rendering, likely due to the program using only one thread. a video with 10000 frames at 60 frames per second takes over 45 minutes to render.

in the screenshot below is in classic task manager on windows 10 describing the issue. note that only 1 of the 12 threads is being harnessed, leading to prolonged render time. is multi-core rendering a possible option?
![image_2024-06-23_184830038](https://github.com/JargeZ/ntscqt/assets/68083196/b1b3f275-d3f8-4b10-8b8f-80614d618457)

**Comment by marine025** [comment]
we need solution, it should be a solution, if vhs camcorder can do this should be able to do it, by releasing a new version or by assignating it more cpu i dont know how

**Comment by marine025** [comment]
this app is my life i need to find a way to increase speed

**Comment by BugBerry202** [comment]
@marine025 begging isn't gonna help

**Comment by valadaptive** [comment]
Does [ntsc-rs](https://github.com/valadaptive/ntsc-rs) work for your use case?

**Comment by marine025** [comment]
yes but it took days or a day of rendering even at 600p resolution videos of 1 hour, with near 20 gb 1080p

will reduce files size to near 2 gb increase speed? with the same time and resolution

**Comment by marine025** [comment]
what is 2x output by nearest neighbor?

**Comment by valadaptive** [comment]
> yes but it took days or a day of rendering even at 600p resolution videos of 1 hour, with near 20 gb 1080p

With ntsc-rs? Can you send a screenshot of it with the estimated render time remaining in the Render tab?

It should definitely not be slower than ntscQT.

**Comment by sakhalif197** [comment]
> Does [ntsc-rs](https://github.com/valadaptive/ntsc-rs) work for your use case?

hey, your software didn't work in my computer
